### DISCUSSION TITLE
Move from Electron to Chrome Headless

### DISCUSSION THREAD

**Discussion by andymsuk** [bbc/bbc-a11y#369] [Feature Requests]
## Summary
Electron is currently used as the browser environment for running the standard tests. Electron has several quirks and currently is the cause of us having to maintain a Docker image for running bbc-a11y on certain environments.

I propose moving away from Electron and using Chrome Headless instead.

**Comment by EmmaJP** [comment]
Andy, assigning this to you temporarily, for investigation into what this change would actually involve in terms of work.

**Comment by andymsuk** [comment]
Have not written/changed any code yet, but from looking at the code and from remembering chats with @joshski, it *should* just be a case of working `./bin/bbc-a11y.js` and all the stuff in `electron` folder to use Chrome instead of Electron. 
Once again, it's not a quick task.

**Comment by oluoluoxenfree** [comment]
So far investigating options, we seem to have a choice between [puppeteer](https://developers.google.com/web/tools/puppeteer/), [chrome remote launcher](https://www.npmjs.com/package/chrome-remote-interface) and [chrome launcher](https://github.com/GoogleChrome/chrome-launcher).
